### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 11

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#480] [Esami & Esoneri M-Z]
### Traccia

<img width="915" alt="image" src="https://github.com/user-attachments/assets/cc972c87-af46-4d1f-873a-d2bd2c93b11e">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/3f0390b1-b326-4993-ab65-870715808c8a)
